spec/usage_spec.lua

```diff
@@ -169,10 +169,38 @@ describe('usage', function()
     end)
   end)
 
-  it("encode_format", function()
-    local usage = Usage.new()
-    usage:add('hits', 0)
-    assert.are.same(usage:encoded_format(), "usage%5Bhits%5D=0")
+  describe("encode format", function()
+    it("works with a single metric", function()
+      local usage = Usage.new()
+      usage:add('hits', 0)
+      assert.are.same(usage:encoded_format(), "usage%5Bhits%5D=0")
+    end)
+
+    it("with multiple metrics return the same", function()
+      local usage = Usage.new()
+      usage:add('a', 0)
```

Comment:
It'd be better to give different values to the metrics to test that they are assigned properly in the resulting string.